snippets/card-collection.liquid

```diff
@@ -75,6 +76,13 @@
           </div>
         </div>
       {%- endif -%}
+      {%- if placeholder_image -%}
+        {{ placeholder_image | placeholder_svg_tag: 'placeholder-svg' }}
+      {%- else -%}
+        {{ 'collection-apparel-4' | placeholder_svg_tag: 'placeholder-svg' }}
+      {%- endif -%}
```

Comment:
Similar thing here. If the collection doesn't have an image we're showing the placeholder instead of the collection picked without an image: 
![](https://screenshot.click/23-17-8emu7-20xxy.png)

I wonder if we could do something like `{{ placeholder_image | default: 'collection-apparel-4' | placeholder_svg_tag: 'placeholder-svg' }}` to simplify things 🤔 

Though I don't think it's currently outputting the image as expected. I can see two scenarios in which the hard coded `collection-apparel-4` should be showing up: 
- cart-drawer.liquid, where it comes up with the issue that I mentioned above for a collection without image
- main-list-collection.liquid, here I don't see the svg coming up, just a grey square with the placeholder collection name instead of the actual collection name ([screenshot](https://screenshot.click/23-23-pzb89-jhpkg.png))